templates/statistics.html

```diff
@@ -79,10 +79,47 @@ <h2>Coverage overview</h2>
     <p>In the table below we have listed the types of organizations that are covered for each country. The amount of information about those organizations and contact points for them varies and is also improved over time.</p>
 
     <table>
-        {{- range . -}}
         <tr>
```

Comment:
You would want to wrap this in a <thead> element, see our other tables for examples.

Reply:
I don't fully get it. It is wrapped by the `tr`, right?

Reply:
Oh, no, it's not. I see now!